tests/javascript/validate/spec.js

```diff
@@ -199,5 +212,34 @@ define(['jquery', 'testsRoot/validate/spec-setup', 'jasmineJquery'], function ($
 				document.getElementById('button').click();
 			});
 		});
+
+		it('removeMarking method on message validation', function () {
```

Comment:
this should be a `describe` call isnt it?

Reply:
Yes. Updated it.